src/fides/api/api/v1/endpoints/saas_config_endpoints.py

```diff
@@ -394,3 +397,80 @@ def register_custom_connector_template(
     return JSONResponse(
         content={"message": "Connector template successfully registered."}
     )
+
+
+@router.post(
+    UPDATE_CUSTOM_TEMPLATE_TO_FILE_TEMPLATE,
```

Comment:
I think this makes more sense as `DELETE /connector_template/{saas_connector_type}`. Also for any user facing text, let's use the term "Fides-provided" or whatever else makes sense to you for the default connector template. The fact that it's a file template is more of an internal implementation detail.